bpf/tests/Makefile

```diff
@@ -7,19 +7,12 @@ MAKEFLAGS += -r
 
 CLANG ?= clang
 
-FLAGS := -I$(ROOT_DIR)/bpf -I$(ROOT_DIR)/bpf/include -O2 -g
-
-CLANG_FLAGS := ${FLAGS} --target=bpf -std=gnu99 -nostdinc
-# eBPF verifier enforces unaligned access checks where necessary, so don't
-# let clang complain too early.
-CLANG_FLAGS += -Wall -Wextra -Werror -Wshadow
-CLANG_FLAGS += -Wno-address-of-packed-member
-CLANG_FLAGS += -Wno-unknown-warning-option
-CLANG_FLAGS += -Wno-gnu-variable-sized-type-not-at-end
-CLANG_FLAGS += -Wimplicit-int-conversion -Wenum-conversion
-CLANG_FLAGS += -Wimplicit-fallthrough
-# Create dependency files for each .o file.
-CLANG_FLAGS += -MD
+FLAGS := -I$(ROOT_DIR)/bpf -I$(ROOT_DIR)/bpf/include -g
+
+# Base CFLAGS (CLANG_FLAGS) are read from pkg/datapath/loader/compile.go
+# to keep them synced with the loader
+CLANG_FLAGS := $(FLAGS) $(shell go run $(ROOT_DIR)/pkg/datapath/loader/tools/clang_cflags.go)
```

Comment:
```suggestion
CLANG_FLAGS := $(FLAGS) $(shell $(GO) run $(ROOT_DIR)/pkg/datapath/loader/tools/clang_cflags.go)
```

Reply:
Fixed, and amended initial commit with your signed off.